site/in-dev/authorization.md

```diff
@@ -75,32 +75,46 @@ Rule definitions are of the form `nessie.server.authorization.rules.<ruleId>=<ru
 
 `<rule_expression>` is basically a CEL expression string, which allows lots of flexibility on a given set of variables. 
 
-Available variables within the `<rule_expression>` are: **'op'** / **'role'** / **'ref'** / **'path'**.
+Available variables within the `<rule_expression>` are: **'op'** / **'role'** / **'roles'** / **'ref'** / **'path'** / **'contentType'**.
 
-* The **'op'** variable in the `<rule_expression>` refers to the type of operation can be any of the following.
+The **'op'** variable in the `<rule_expression>` refers to the type of operation can be any of the following.
   See [BatchAccessChecker](https://github.com/projectnessie/nessie/blob/main/servers/services/src/main/java/org/projectnessie/services/authz/BatchAccessChecker.java)
-  and [Check](https://github.com/projectnessie/nessie/blob/main/servers/services/src/main/java/org/projectnessie/services/authz/Check.java) types.
-  * Arguments to the CEL script for the following check-types:
+  and [Check](https://github.com/projectnessie/nessie/blob/main/servers/services/src/main/java/org/projectnessie/services/authz/Check.java) types for details.
+
+* Checks for Reference operations:
```

Comment:
```suggestion
#### Checks for Reference operations

```

Reply:
And unindent the following lists (same below)

Reply:
Thatnks for the tip.... In the end I ended up re-formatting this text quite a bit... I hope for the better :)